src/dmd/cparse.d

```diff
@@ -82,6 +83,15 @@ final class CParser(AST) : Parser!AST
                 auto ld = new AST.LinkDeclaration(token.loc, LINK.c, symbols);
                 wrap.push(ld);
 
+                if (importBuiltins)
+                {
+                    /* Seen references to C builtin functions.
+                     * Import their definitions
+                     */
+                    auto s = new AST.Import(Loc.initial, null, Id.builtins, null, false);
+                    wrap.push(s);
```

Comment:
Wouldn't it be easier to do this import regardless of whether we have found compiler extensions or not? That would simplify the implementation at the cost of having at useless import.

Reply:
I thought about that, and decided against it because of compile speed.

Reply:
The extra test here (2 instructions per file) will not be detectable in a benchmark, but importing __builtins, object, stdarg, etc., will show up.